tests/selenium_tests/new_excerpt.py

```
@@ -11,3 +11,21 @@ def new_excerpt(driver, base_url):
     # choose an area in monaco (North = 43.734716500825 | East = 7.42564201354981
     #                           | South = 43.7289719167851 | West = 7.41568565368652)
     geojson_field.send_keys('{"type":"Polygon","coordinates":[[[7.423303127288818,43.7380498124601],[7.418947219848633,43.73685604310849],[7.418625354766846,43.73373972784547],[7.420299053192139,43.73046819754524],[7.425191402435303,43.72959989409132],[7.428925037384032,43.72961539962052],[7.432680130004883,43.73491805517113],[7.433323860168456,43.737662227617484],[7.428624629974364,43.74053029604388],[7.422015666961669,43.74063881485443],[7.42100715637207,43.74180150405227],[7.418088912963867,43.741817006422366],[7.423303127288818,43.7380498124601]]]}')
+
+
+def new_excerpt_through_admin(selenium, base_url, excerpt_name="existing_excerpt"):
+    selenium.get('{0}/admin/excerptexport/excerpt/'.format(base_url))
+    try:
+        selenium.find_element_by_xpath("//a[contains(text(),'{}')]".format(excerpt_name))
+        return
+    except:
```

consider only swallowing a more specific error